crates/ursa-pod/fuzz/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "ursa-pod-fuzz"
```

Comment:
can we add this to `ursa-pod` Cargo.toml workspace?

Reply:
no, this is a standard `cargo fuzz` subproject created by `cargo fuzz init`, checkout other repos.